src/server/tsconfig.json

```diff
@@ -2,7 +2,7 @@
 {
   "extends": "../../tools/tsconfig-base.json",
   "compilerOptions": {
-    "lib": ["ES2023"],
+    "lib": ["ES2022"],
```

Comment:
Moved because of this on esbuild. If someone knows how to fix I'm happy to change it back!

```
▲ [WARNING] Unrecognized target environment "ES2023" [tsconfig.json]

    ../../tools/tsconfig-base.json:40:14:
      40 │     "target": "ES2023"
```